PoC-differ/differ.go

```diff
@@ -305,7 +305,7 @@ func processAllReportsFromCurrentWeek(ruleContent map[string]types.RuleContent,
 			Int("important notifications", digest.ImportantNotifications).
 			Msg("Producing weekly notification for ")
 
-		notification := generateWeeklyNotificationMessage(notificationConfig.InsightsAdvisorURL, string(account), digest)
+		notification := generateWeeklyNotificationMessage(notificationConfig.InsightsAdvisorURL, fmt.Sprint(account), digest)
```

Comment:
dtto

Reply:
I'm thinking it's probably not needed, as in the final step I Marshall it all into a string. But if you don't mind, I'd keep this one like this until I add the generateWeeklyNotificationMessage UT, for "secure refactoring" =). WDYT?

Reply:
ok